test/e2e/live_iso_test.go

```diff
@@ -91,6 +91,7 @@ func liveIsoTest() {
 		Expect(err).NotTo(HaveOccurred(), "Error getting all BMHs")
 
 		for _, bmh := range bmhs {
+			bmh := bmh
```

Comment:
Is this calming the lint

Reply:
Yes, it is for gosec G601: "Implicit memory aliasing of items from a range statement".
Ref: https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop

Reply:
```suggestion
			bmh := bmh # prevent gosec G601
```

Reply:
Done